brain/python/arrayHelpers.cpp

```diff
@@ -239,6 +239,52 @@ bp::object toNumpy( const brain::Matrix4f& matrix )
     return bp::object( bp::handle<>( array ));
 }
 
+
+bp::object frameToTuple(CompartmentReportFrame&& frame)
+{
+    return bp::make_tuple(frame.getTimestamp(), toNumpy(frame.takeData()));
+}
+
+bp::object framesToTuple(CompartmentReportFrames&& frames)
+{
+    if(frames.empty())
+        return bp::make_tuple(-1.f,std::vector<float>());
+    if(frames.size() == 1)
+        return frameToTuple(std::move(frames.front()));
```

Comment:
This violates the API contract. Even if the list has a single element, this function has still to return a list of timestamps and and a 2D matrix. Otherwise the users have to check for this particular case whenever calling load(start, end) or loadAll(), otherwise their Python code may raise exceptions.